scripts/ci.sh

```diff
@@ -65,6 +65,8 @@ setup_mf() {
         fi
     done
     make -j$NPROC
+    echo "Compile check for rabbitmq..."
+    MF_BROKER_TYPE=rabbitmq make -j $NPROC http
```

Comment:
Put this before `make -j $NPROC`: https://github.com/mainflux/mainflux/pull/1570#discussion_r901527315